Dockerfile

```diff
@@ -0,0 +1,35 @@
+FROM debian:trixie
+
+RUN apt-get update && apt-get upgrade -y
```

Comment:
not necessarily: the next lines might run months later due to caching, at which point the update is out of date and apt installs won't work - that happens a lot to me.

All of the apt lines could all move together though into one line. The historical reason they're here is because in the environment I stole this dockerfile from, I wanted to cache intermediate steps while fiddling with later steps a lot. If that fiddling isn't happening much, it can all go into a single update/upgrade/install `RUN`

Reply:
Got it; makes sense.